plugins/kubernetes/src/components/PodExecTerminal/PodExecTerminalAttachAddon.ts

```diff
@@ -30,7 +30,7 @@ export class PodExecTerminalAttachAddon extends AttachAddon {
         return;
       }
 
-      const buffer = Uint8Array.from([0, ...textEncoder.encode(data)]);
+      const buffer = Uint8Array.from([0, ...this.#textEncoder.encode(data)]);
```

Comment:
What's the need for a local variable in the class? What difference does it make compared to the scoped constant at the top of the file?

Reply:
Having it at the root of the file makes it get executed _instantly_ when the module is imported, no matter if the class is used or not. Having it inside the class, means that the `TextEncoder` only gets instantiated when the class gets instantiated (which does NOT happen inside `packages/app` test runs).

Reply:
That's actually a big difference, thanks for clarifying.